package.json

```diff
@@ -98,16 +97,16 @@
     "zen-observable": "^0.6.0"
   },
   "scripts": {
-    "all": "npm run lint && npm run flow && npm run test && npm run build && npm run size",
+    "all": "npm run lint && npm run typecheck && npm run test && npm run build",
     "build": "npm run clean && npm run build:lib && npm run build:umd && npm run build:tests",
-    "build:lib": "babel src --out-dir lib && npm run flow:copy-src",
+    "build:lib": "babel src --out-dir lib && npm run build:flow",
     "build:umd": "rollup -c",
     "build:tests": "cross-env NODE_ENV=test webpack tests/index.js tmp/tests.js",
+    "build:flow": "echo \"// @flow\n\nexport * from '../src';\" > lib/index.js.flow",
```

Comment:
wait, does echo work on windows?

Reply:
It always worked for me there. For my previous coworkers on windows this way also worked afaik.